.github/workflows/functional-test.yml

```diff
@@ -196,3 +196,145 @@ jobs:
       with:
         name: appium-android-${{matrix.test_targets.name}}.log
         path: appium.log
+
+  flutter_e2e_test:
```

Comment:
If the unit test was sufficient as this client-side implementation's test, this `flutter_e2e_test` can be removed from this PR as redundant slower tests

Reply:
Flutter e2e tests are executed in parallel with others. They should not slow down the main flow.

Reply:
@KazuCocoa As mentioned by @mykola-mokhnach  All e2e tests executed in parallel. Also having e2e tests would be easy for someone to figure out how to use the driver/flutter methods.

Reply:
Sure.

My intention was if both had the same purpose to check the same python client functionality, e2e just took longer time than the unit test and flakiness, so just it is redundant as checking.

It's fine to keep it. Then, could you leave somewhere when this e2e has an issue, who could we mention...? In `flutter_e2e_test` section is fine. (or just mention MummanaSubramanya?) Main maintainers in python client have no knowledge about the drier. We may need to mention you when this e2e driver fails if such failure is the driver side issue or client.

For example usage, I'd recommend you have such example in the driver project and run it every driver update to detect possible changes in the driver side that effects client implementation. Not main topic here but more general thing

Reply:
@KazuCocoa I completely agree with your concern about driver issue. As you suggested I can mention my name in the workflow file.

Agreed with your recommendation to have the e2e tests in a seperate standalone project. I can do it in a different PR if you want.

Reply:
thank you for the comment :)